### ISSUE TITLE
BUG: Main Button, E-Brake and throttle became unresponsive

### ISSUE THREAD

**Issue by TastyTom79** [scooterhacking/scooterhacking-issues-public#113]
### Scooterhacking Tool

Firmware (SHFW)

### Type of issue

Other

### Severity of issue

medium (affects usability but has a workaround)

### SHUT Version

SHUT 2.6.1

### SHFW Version

SHFW 3.9.1

### Scooter model

Ninebot G2

### Description of the Bug

After riding the scooter with the settings mentioned in step to reproduce, most of the scooter "headset" became unresponsive.

### Steps to Reproduce

Settings I used -- G2 SHFW guide by KingBro
!!Enable Expert view in the top right corner of the Config!!
Throttle
• Set a 25A DPC Curve. 
• 100% acceleration boost.
• Overmodulation ON
Brake
• Set a 55A Curve.
• Set Brake boost between 20-60% depending on your weight.
Field Weakening
• Start speed: 20kmh
• Initial field current: 0A
• Variable field current: 2000mA/km/h
• Iq: 75A
• Id: 45A
Motor settings
Wheel size: 9.7 inches for accurate dash speed

I was riding for 1-2 kilometers, then while waiting on a traffic light 
I wanted to accelerate -> nothing happend 
I wanted to reboot scooter -> main button didnt work
E-Brake also didn't work

Workaournd:
- Wait for the scooter to turn itself of
- And boot it again

Turn Signal and E-Horn did work tho.

### Expected vs. Actual Behavior

Expected:
- Scooter shall work as expected

Actual Behavior:
- I wanted to accelerate -> nothing happend 
- I wanted to reboot scooter -> main button didnt work
- E-Brake also didn't work

### Contact Information

tastytom - Discord

**Comment by TastyTom79** [comment]
Edit:

I found out why, it went into to locked mode as I played with different settings, probably far fingered.

I experienced now the same with the locking functionality of SHUt

**Comment by TastyTom79** [comment]
Is caused by locking the scooter, will open another issue.
